include/library/spdm_requester_lib.h

```diff
@@ -491,6 +492,19 @@ libspdm_return_t libspdm_start_session(void *spdm_context, bool use_psk,
                                        uint8_t *heartbeat_period,
                                        void *measurement_hash);
 
+/**
+ * Sends the `GET_VERSION` and `GET_CAPABILITIES` requests, where GET_CAPABILITIES.Param1[0] is set
+ * If the Responder supports this extended capability, the Responder will include the Supported
+ * Algorithms Block in its CAPABILITIES response.
+ *
+ * @param spdm_context The SPDM context.
+ * @param responder_supported_algorithms The responder supported algorithms.
+ * @retval RETURN_SUCCESS               The connection is initialized successfully.
+ * @retval RETURN_DEVICE_ERROR          A device error occurs when communicates with the device.
+ **/
+libspdm_return_t libspdm_get_supported_algorithms(
+    void *spdm_context, libspdm_responder_supported_algorithms_t *responder_supported_algorithms);
```

Comment:
Need to use `void *` for responder_supported_algorithms, because the structure may be changed in the future version.
Also add length for the function.

Reply:
See https://github.com/DMTF/libspdm/pull/2968#discussion_r2015409783